kolibri/core/discovery/utils/filesystem/posix.py

```diff
@@ -0,0 +1,177 @@
+import logging
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+from .constants import drivetypes
+
+logger = logging.getLogger(__name__)
+
+MOUNT_PARSER = re.compile("^(?P<device>\S+) on (?P<path>.+?) ((type (?P<filesystem1>\S+)|(\((?P<filesystem2>[^, ]+))))", flags=re.MULTILINE)
```

Comment:
Removed and replaced with two separate and (subjectively) highly readable regexes, one for OSX and one for Linux. Besser?